src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService_FindDeclaration.cs

```diff
@@ -108,6 +108,23 @@ public bool CanAddTo(SyntaxNode destination, Solution solution, CancellationToke
             return availableIndices != null && availableIndices.Any(b => b);
```

Comment:
the PR shows a 32k change to ServicesVSResources.resources.  Why was that changed?

Reply:
I don't know why my VS wants to change the resources file. I will reset them back